docs/operations-guide/running-metabase-on-elastic-beanstalk.md

```diff
@@ -102,7 +102,7 @@ Once there, enter a database username and password. We suggest you hold onto thi
 Regarding individual settings, we recommend:
 
 * `Snapshot` should be left as `None`.
-* `DB engine` should be set to `postgres`. Metabase also supports MySQL/Maria DB as backing databases, but this guide currently only covers running Metabase on Postgres.
+* `DB engine` should be set to `postgres`. Metabase also supports MySQL/Maria DB as backing databases, but this guide currently only covers running Metabase on Postgres (i.e just changing the DB engine to mysql will cause a crash when Metabase starts because he postgres driver will be used to connect to a mysql instance...)
```

Comment:
Why is the `postgres` driver being used to connect to `mysql`?

Reply:
Well that was the bug I had experienced at the time (probably because the deployment script does not adapt to mysql and postgres jdbc driver is hardcoded somewhere)